packages/cli/src/commandsConfig.ts

```diff
@@ -414,7 +414,13 @@ const commandsConfig = {
         description: 'IPFS hash to write deployment data for',
       },
     ],
-    options: [...debugVerbosity],
+    options: [
```

Comment:
Added chain id option to pin command because the pin command fails when trying to pin a package that was built on a live network